src/CodeAnalyzers/Bit.CodeAnalyzers/SystemAnalyzers/MethodAsyncVoidAnalyzer.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.CSharp;
+using System;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Bit.CodeAnalyzers.SystemAnalyzers;
+
+[DiagnosticAnalyzer(LanguageNames.CSharp)]
+public class MethodAsyncVoidAnalyzer : DiagnosticAnalyzer
+{
+    public const string DiagnosticId = nameof(MethodAsyncVoidAnalyzer);
+
+    public const string Title = "Change return type of async method to Task";
+    public const string Message = "Return type of async method must be Task or contain try/catch";
```

Comment:
please change this value to this:
`Either change the return type to 'Task' or use try/catch to handle the exceptions.`